maestro-cli/src/main/java/maestro/cli/command/UploadCommand.kt

```diff
@@ -48,34 +48,37 @@ import kotlin.system.exitProcess
 )
 class UploadCommand : Callable<Int> {
 
-    @CommandLine.Parameters
+    @CommandLine.Parameters(description = ["The name of this upload"])
     private lateinit var uploadName: String
 
-    @CommandLine.Parameters
+    @CommandLine.Parameters(description = ["App binary to run your Flows against"])
     private lateinit var appFile: File
 
-    @CommandLine.Parameters
+    @CommandLine.Parameters(description = ["Flow file or directory"])
     private lateinit var flowFile: File
 
-    @Option(names = ["--apiKey"])
+    @Option(order = 0, names = ["--apiKey"], description = ["API key"])
     private var apiKey: String? = null
 
-    @Option(names = ["--apiUrl"])
+    @Option(order = 1, names = ["--apiUrl"], description = ["API base URL"])
     private var apiUrl: String = "https://api.mobile.dev"
 
-    @Option(names = ["--repoOwner"])
+    @Option(order = 1, names = ["--mapping"], description = ["(Android-only) Proguard mapping file"])
```

Comment:
We also support dsym file which will be important for memory leak, and crash symbolication. I think let's include that as well.

Reply:
Ah yep good catch